### ISSUE TITLE
Feature wish: Display mastodon link on profile page with rel=me

### ISSUE THREAD

**Issue by schtobia** [nextcloud/integration_mastodon#36]
As a normal user, I want to be able to display my Mastodon profile on my Nextcloud profile page with the "rel" attribute set to "me", so that I can use my Nextcloud profile link as a verified link on mastodon (double sided link).

**Comment by julien-nc** [comment]
Could you elaborate on that?
Do you want the link generated in your NC profile to have the "rel" attribute set to "me"?

**Comment by schtobia** [comment]
exactly. Mastodon verifies a submitted by checking if the source has a link back to the Mastodon profile with the "rel" attibute set: https://docs.joinmastodon.org/user/profile/#verification .

**Comment by janepie** [comment]
As this would have to be done in the profile settings, I am moving this to the server repository :)
